pointgrey_camera_driver/launch/stereo.launch

```diff
@@ -0,0 +1,61 @@
+<launch>
+  <!-- Common parameters -->
+  <arg name="camera_name" default="camera" />
+  <arg name="frame_rate" default="15" />
+
+  <arg name="left_camera_serial" default="0" />
+  <arg name="left_camera_calibrated" default="0" />
+
+  <arg name="right_camera_serial" default="0" />
+  <arg name="right_camera_calibrated" default="0" />
+
+  <group ns="$(arg camera_name)" >
+    <group ns="left" >
+      <node pkg="nodelet" type="nodelet" name="left_camera_nodelet_manager" args="manager" />
```

Comment:
Couldn't both cameras go in the same nodelet manager, or does that cause namespacing chaos?